music21/midi/translate.py

```diff
@@ -1198,25 +1198,22 @@ def elementToMidiEventList(
         # TODO: skip Harmony unless showAsChord
         sub = chordToMidiEvents(el, includeDeltaTime=False)
     elif 'Dynamic' in classes:
-        return  # dynamics have already been applied to notes
+        return None  # dynamics have already been applied to notes
     elif 'TimeSignature' in classes:
         # return a pair of events
-        el: 'music21.meter.TimeSignature'
```

Comment:
Indeed it does give a warning now.  It might be that we need to import these modules and use a slower set of isinstance() checks or assert isinstance rather than class checking to make this work.

Reply:
Ah, then we have contradictory linters! Mine always underlines it as invalid because it's not a TypedDict. I'll have a look and see what the precise message was or what mypy said.

Reply:
```shell
$ python -m mypy --follow-imports=skip music21/midi/translate.py 
music21/midi/translate.py:1211: error: Name "el" already defined on line 1178
music21/midi/translate.py:2484: error: Invalid type comment or annotation
```

Reply:
I can put them back in so they don't annoy anybody, since we're not running mypy over midi yet. But good that we flushed out this discrepancy. Haven't looked into whether it depends on a specific version of mypy.